defs.bzl

```diff
@@ -41,19 +41,6 @@ npm_package = _npm_package
 npm_package_bin = _npm_bin
 # ANY RULES ADDED HERE SHOULD BE DOCUMENTED, see index.for_docs.bzl
 
-# Allows us to avoid a transitive dependency on bazel_skylib from leaking to users
-def dummy_bzl_library(name, **kwargs):
-    native.filegroup(name = name)
-
-# @unsorted-dict-items
-COMMON_REPLACEMENTS = {
-    # Replace loads from @bazel_skylib with the dummy rule above
-    "(load\\(\"@bazel_skylib//:bzl_library.bzl\", \"bzl_library\"\\))": "# bazel_skylib mocked out\n# $1\nload(\"@build_bazel_rules_nodejs//:defs.bzl\", bzl_library = \"dummy_bzl_library\")",
-    # Cleanup up package.json @bazel/foobar package deps for published packages:
-    # "@bazel/foobar": "file:///..." => "@bazel/foobar": "0.0.0-PLACEHOLDER"
-    "\"@bazel/([a-zA-Z_-]+)\":\\s+\"(file|bazel)[^\"]+\"": "\"@bazel/$1\": \"0.0.0-PLACEHOLDER\"",
-}
-
 def node_modules_filegroup(packages, patterns = [], **kwargs):
     native.filegroup(
         srcs = native.glob(["/".join([
```

Comment:
should also remove VERSION & BAZEL_VERSION from this file?

Reply:
done